values.yaml

```diff
@@ -126,7 +127,9 @@ global:
 
   dagOnlyDeployment:
     enabled: false
-    image: quay.io/astronomer/ap-dag-deploy:0.5.3
+    images:
+      repository: quay.io/astronomer/ap-dag-deploy
+      tag: 0.5.3
```

Comment:
we actually dont need images we can just keep it simple like
```
dagOnlyDeployment:
    enabled: false
    repository: quay.io/astronomer/ap-dag-deploy
    tag: 0.5.3
```

Reply:
@danielhoherd what do you think if you are okay with above format i'm fine

Reply:
Sure, will do the same, I thought that we need to keep it similar to what we have at other places like: https://github.com/astronomer/astronomer/blob/3b3a9481295fe95d7074d3529a2d1c7a3d5835de/charts/alertmanager/values.yaml#L9C1-L13C29

Reply:
https://github.com/astronomer/astronomer/blob/master/charts/astronomer/templates/houston/houston-configmap.yaml#L152-L158

we are internally parsing the format from our configmap i feel like global can be real simple initial ask was to have repository and tag as seperate just a thought here

Reply:
you need to fix in the configmap as well since i have added a split list directly in the config

Reply:
FWIW there is an issue to standardize the way we configure images so that they are all the same data structure. It might be worth thinking about that when we think about what to do here.

Reply:
How could we make this so that changing one value enables an airgap? For instance, how could we make the `quay.io/astronomer` part a global value, and we only have `repo: ap-vector`, `tag: 1.2.3` ?

Reply:
@pgvishnuram config map for fixed, can you check and let me know if there is anything else.?

Reply:
I have fixed failing ones i have some doubt on formats imposed at global level we can discuss on the standup to finalize and merge the changes